app/views/tradingpremises/remove_trading_premises.scala.html

```diff
@@ -23,40 +23,34 @@
 @header = {
     @components.back_link()
     @errorSummary(f)
-    @if(showDateField) {
-        @heading("tradingpremises.remove.trading.premises.enddate.lbl", "summary.tradingpremises")
-    } else {
+    @if(!showDateField) {
         @heading("tradingpremises.remove.trading.premises.title", "summary.tradingpremises")
     }
 }
 
+@content = {
```

Comment:
Any reason this is content and not supporting content?

Reply:
Because there is an if/else condition where this is the only content displayed (Line 53). I'm happy to rename it to whatever is felt better.